data_registry/process_manager/task/process.py

```diff
@@ -16,7 +16,7 @@ def url_for_collection(*parts):
 class Process(TaskManager):
     def __init__(self, job):
         self.job = job
-        self.process_id = job.context["process_id"]
+        self.process_id = job.context.get("process_id", None)
```

Comment:
Why was this change needed?

Reply:
Aha, right, when these classes are initialized only to run wipe(). I'll see if there's a better way.

Reply:
Okay, I think I fixed that logic issue correctly now. Can you test? (Ideally we would have an automated test as well)

Reply:
Now it works